packages/ListBox/src/components/Filter/effects.js

```diff
@@ -0,0 +1,24 @@
+export const handleEffectValue = (props, applyFilter) => () => {
```

Comment:
If all we care about is `value`, wouldn't it be better to accept that as the argument instead of the entire `props` object?